app/views/images/index.html.erb

```diff
@@ -1,13 +1,13 @@
-<%= stylesheet_link_tag "image" %>
-
-<h1>Welcome!</h1>
-
-<%= link_to 'New image', new_image_path %>
-
-<table class="image_table">
-  <% @images.each do |image| %>
-      <tr>
-        <td><%= image_tag image.imageurl, class: "index_image" %></td>
-      </tr>
-  <% end %>
-</table>
+<div class="album">
+  <div class="container-fluid">
+    <div class="row align-items-center">
+      <% @images.each do |image| %>
+        <a href=<%= "#{images_path}/#{image.id}" %>>
+          <div class="card">
```

Comment:
In html, its not a good practice to include a div inside of an anchor (a) tag. Get rid of that div and add the class card to the div with the row class

Reply:
As mentioned above, I meant for the whole card to be clickable, and not just the image.  I found that by not including the div within the anchor, the link behavior didn't feel right.  I wanted to be able to click anywhere in the card to show the image, not just within the bounds of the image itself -- it felt like a more natural interaction model.